en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configurations/adapter-configurations.md

```diff
@@ -602,4 +602,256 @@ See the example .toml file given below.
     </section>
 </div>
 
+## Source Control
```

Comment:
@nandulaperera , please use the config catalog generator to add the new configs so that the next time someone uses the Config catalog generator your changes will not get wipes out

Reply:
Fixed with 948474e